services/cgr-engine.go

```diff
@@ -119,6 +119,10 @@ func (cgr *CGREngine) GetServDeps() map[string]*sync.WaitGroup {
 	return cgr.srvDep
 }
 
+func (cgr *CGREngine) GetDbService() *DataDBService {
```

Comment:
Do we still need this method now that we are not longer using dataDB outside of engine?